frame/nis/src/lib.rs

```diff
@@ -704,35 +760,176 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Reduce or remove an outstanding receipt, placing the according proportion of funds into
+		/// the account of the owner.
+		///
+		/// - `origin`: Must be Signed and the account must be the owner of the receipt `index` as
```

Comment:
I dont quite get `"and the account must be the owner of the receipt"` since communal receipts dont have an owner IIUC.